content/geometry/geometry-index.adoc

```diff
@@ -0,0 +1,65 @@
+= Vehicle structure
+:toc:
+:toclevels: 4
+
+
+include::vehicle-structure/introduction.adoc[leveloffset=+1]
```

Comment:
Do we actually need individual files for all the elements? What is the benefit compared to putting all element descriptions in one file "vehicle-structure.adoc"?

Reply:
This also contradicts the general structure of the Geometry chapter introduced in #36, in my opinion.

Reply:
Individual files for all elements have the following advantages:
- Information on a certain topic are easier to find in the file structure
- Group members can work on different topics at the same time without interference and possible merge conflicts 
- Changes can be tracked more easily
- Topics can be moved more easily in the file structure or even between chapters
- Linking to individual topics is easier in AsciiDoc than linking to subsections 

We will have to align on how the chapter structure of the document and of the subchapters (e.g. geometry) looks like.

Reply:
Group consensus: But topics in one file